packages/suite-data/tsconfig.json

```diff
@@ -1,6 +1,7 @@
 {
     "extends": "../../tsconfig.json",
     "compilerOptions": {
+        "target": "es5",
```

Comment:
Without `target: es5`, there are `=>` and `const` in compiled code.

Reply:
You complied somewhere else than `libDev`? it doesn't matter what is in `libDev` that folder is just temp folder used in typecheck and this particular tsconfig shouldn't be used for anything else that type-check here

Reply:
```
    entry: path.resolve(__dirname, './src/browser-detection/index.ts'),
    output: {
        path: path.resolve(__dirname, 'files/browser-detection'),
        filename: 'index.js',
    },
```

Reply:
I see, it is using `ts-loader` in webpack, so thats reason why it is using `tsconfig.ts` but that's not good because we should not use TS for transpiling, we should use babel as we are doing in suite etc. We should migrate from `ts-loader` to `babel-loader` which we using in suite and which we should use everywhere. You can try something like this:

```
{
      test: /\.ts$/,
      exclude: /node_modules/,
      use: {
        loader: 'babel-loader',
        options: {
          presets: [
            ['@babel/preset-env', { targets: "defaults" }]
            '@babel/preset-typescript'
          ]
        }
      }
    }
```

maybe you will need to tune `preset-env` target, ideally to same browser config you are using here

Reply:
93af2807ca3a717b2f505221b2d3d90407d9df4e